plugins/code-review/commands/pr.md

```
@@ -0,0 +1,206 @@
+---
+description: "Automated PR code quality review with language-aware analysis and project-specific profiles"
+argument-hint: "<pr-url-or-number> [--language <lang>] [--profile <name>] [--skip-build] [--skip-tests]"
+---
+
+## Name
+code-review:pr
+
+## Synopsis
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)